runner/execer/os/os_execer.go

```diff
@@ -2,11 +2,10 @@ package os
 
 import (
 	"errors"
+	"github.com/scootdev/scoot/runner/execer"
```

Comment:
We've been segregating imports. So stdlib are in one block, then scoot in another.

Is there a reason to change?